xpdacq/xpdacq.py

```diff
@@ -49,7 +49,13 @@ def _update_dark_dict_list(name, doc):
     dark_dict = {'acq_time': acq_time, 'exposure': light_cnt_time,
                  'timestamp': doc['time'], 'uid': doc['run_start']}
     dark_dict_list.append(dark_dict)
-    glbl['_dark_dict_list'] = dark_dict_list  # update glbl._dark_dict_list
+    if doc['exit_status'] == 'success':
+        print('dark frame complete, update dark dict')
+        dark_dict_list.append(dark_dict)
+        glbl['_dark_dict_list'] = dark_dict_list  # update glbl._dark_dict_list
+    else:
+         print("INFO: dark scan was not successfully executed.\n"
```

Comment:
Unfinished header usually happens when the user aborts scan intentionally as suspending would not close the header. To increase the visibility of this error, I am thinking the official python ``warning`` module paired with ``logging``, so that user is aware of error and know where to look at the error logs.